vector/src/main/java/im/vector/app/features/onboarding/OnboardingAction.kt

```diff
@@ -41,6 +41,7 @@ sealed interface OnboardingAction : VectorViewModelAction {
 
     // Login or Register, depending on the signMode
     data class LoginOrRegister(val username: String, val password: String, val initialDeviceName: String) : OnboardingAction
+    data class Register(val username: String, val password: String, val initialDeviceName: String) : OnboardingAction
```

Comment:
100% agree, would favour using register & login.

For now I'll rename the code introduced by the PR rather than renaming the entire `onboarding` package as it will cause a lot of noise in the diff